smf/wicketd/manifest.xml

```diff
@@ -12,7 +12,7 @@
   </dependency>
 
   <exec_method type='method' name='start'
-    exec='ctrun -l child -o noorphan,regent /opt/oxide/wicketd/bin/wicketd run /var/svc/manifest/site/wicketd/config.toml --address %{config/address} --artifact-address %{config/artifact-address} --mgs-address %{config/mgs-address}  &amp;'
+      exec='ctrun -l child -o noorphan,regent /opt/oxide/wicketd/bin/wicketd run /var/svc/manifest/site/wicketd/config.toml --address %{config/address} --artifact-address %{config/artifact-address} --mgs-address %{config/mgs-address} --baseboard-identifier %{config/baseboard-identifier} --baseboard-model %{config/baseboard-model} --baseboard-revision %{config/baseboard-revision} &amp;'
```

Comment:
I'm forgetting where we landed, but just to confirm: Did we determine that this information is not accessible from within a non-global zone?

If possible, it seems massively preferable (from a safety and plumbing point-of-view) for wicketd to ask "what is my baseboard info" rather than relying on the sled agent to plumb it through.

Reply:
This information is indeed not accessible from a non-global zone. I chatted with @citrus-it briefly about making the `devinfo` device accessible in a zone, and it sounded like there were enough questions around permissions/security that it would be more straightforward to have sled-agent plumb this through.